### PR TITLE
Add FOURCC code for Y212 and Y412 format

### PR DESCRIPTION
**Pull request by wangyan42164**
Signed-off-by: Yan Wang <yan.wang@linux.intel.com>

### CODE REVIEW

**Comment by wangyan42164** [comment]
@XinfengZhang 
This PR is for https://github.com/intel/media-driver/issues/998. 
Could you please review it?
Thanks.

**Comment by XinfengZhang** [comment]
LGTM, but I prefer to separate FOURCC to a new file in future

File: va/va.h [hunk 1]
```diff
@@ -4041,6 +4047,12 @@ VAStatus vaSyncBuffer(
  * A, V, Y, U are found in bits 31:30, 29:20, 19:10, 9:0 respectively.
  */
 #define VA_FOURCC_Y410          0x30313459
+/** Y412 packed 12-bit YUVA 4:4:4.
+ *
+ * Each pixel is a set of four samples, each of which is a two-byte little-endian value.
+ * The samples are in the order A, V, Y, U.
```
Comment (XinfengZhang) [inline va/va.h:4053]: @fulinjie how about to define it to X, V, Y U directly, to address concern from ffmpeg community?

Comment (fulinjie) [reply va/va.h:4053]: Yes, community seems to prefer XVYU if Alpha channel is not actually supported.

**Pull request merged**